src/frequenz/client/base/streaming.py

```diff
@@ -24,15 +23,15 @@
 """The output type of the stream."""
 
 
-class GrpcStreamingHelper(Generic[InputT, OutputT]):
+class Broadcaster(Generic[InputT, OutputT]):
```

Comment:
`frequenz.client.base.streaming.Broadcaster` feels somewhat broad, for a thing that takes a streaming grpc call and puts the gprc messages into a channel.  If anything, it should be the name for an ABC, when there's such a need, but not to any one thing.

I like `GrpcStreamBroadcaster`, because for someone who sees it, it already gives some idea what it does.  But maybe there are other options.

Reply:
Grpcaster :eyes:

Reply:
This is the point I expected to be controversial the most, because I wasn't also 100% convinced. I'm OK with `GrpcStreamBroadcaster` even if I find it a bit too verbose, at least in the context of my vision of having this repo renamed to `frequenz-grpcutil` and then using `from frequenz.grpcutil import streaming; b = streaming.GrpcStreamBroadcaster`.

But when in doubt, I definitely think more explicit names are better than more generic ones, so let's go for that. Will update.